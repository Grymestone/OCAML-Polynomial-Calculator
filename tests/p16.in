2a^4 - (2a^1 + 8a^3)^2
