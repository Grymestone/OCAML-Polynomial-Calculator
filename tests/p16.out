((2*(a)^4)-(((2*(a)^1)+(8*(a)^3)))^2)
-30x^4 + -64x^6 + -4x^2